src/main/java/by/teachmeskills/module/book/BookSqlQueries.java

```diff
@@ -0,0 +1,11 @@
+package by.teachmeskills.module.book;
+
+public class BookSqlQueries {
+    public static final String INSERT_BOOK = "INSERT  INTO books (title,author,description,year,price) VALUES(?,?,?,?,?);";
+    public static final String GET_ALL_BOOKS = "SELECT * FROM books";
+    public static final String DELETE_BOOK = "DELETE FROM books WHERE id = ?";
+    public static final String UPDATE_BOOK = "UPDATE books SET price WHERE id = ? ";
```

Comment:
Incorrect update query: should be the following:
_UPDATE books SET price = ? WHERE id = ?_